### ISSUE TITLE
Could not find a Cypress configuration file `cypress.json` on Jenkins

### ISSUE THREAD

**Issue by omelnik** [cypress-io/cypress#9084]
I decided to put my tests into a docker container, but I started facing an issue.
When I tried to map my current dir to a folder inside my container, I started getting this error: `Could not find a Cypress configuration file, exiting. We looked but did not find a cypress.json file in this folder: /usr/src/app`

Here's my Dockerfile:

```sh
# pull official Cypress image
FROM cypress/included:5.5.0

# set working directory
RUN mkdir /usr/src/app
WORKDIR /usr/src/app

# copy our test application
COPY package.json /usr/src/app/package.json
COPY package-lock.json /usr/src/app/package-lock.json

# install NPM dependencies and Cypress binary
RUN npm ci

# copy all files into the container
COPY . /usr/src/app
COPY cypress.json /usr/src/app/cypress.json

# check if the binary was installed successfully
RUN $(npm bin)/cypress verify

# run tests
CMD ["./node_modules/.bin/cypress", "run"]
````

Here's my command
```
docker run -v $PWD:/usr/src/app --ipc=host image:latest --headless --browser chrome
```

And like usually it works on my local, but doesn't work on Jenkins. I'm not sure what I'm missing. I was using [this article](https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/) as the reference. Just for the record, I tried to pass `-w /usr/src/app` flag as well.

**Comment by cdohotaru** [comment]
@omelnik I don't know if this is the case for you but I mapped once the volume wrong. 
Instead of "$PWD:/usr/src/app" i should have had "$PWD**/e2e**:/usr/src/app" as that's where my cypress.json is.

**Comment by ridakejji** [comment]
I am having the same issue and changing the directory doesn't solve it

**Comment by andersonterry** [comment]
[Windows 10], I get the same issue as follows (cypress.json is present in this dir):

docker run -it -v /c/Users/TAnderson2/source/repos/cypress/integration/1-getting-started:/e2e -w /c/Users/TAnderson2/source/repos/cypress/integration/1-getting-started cypress/included:7.7.0

Could not find a Cypress configuration file, exiting.

We looked but did not find a cypress.json file in this folder: /c/Users/TAnderson2/source/repos/cypress/integration/1-getting-started

**Comment by andersonterry** [comment]
did an inspect on the container for the root, and then tried this with a different error:

C:\>docker run -it -v //c/Users/TAnderson2/source/repos/cypress/integration:**/root/.cache/Cypress/7.7.0/Cypress/Cypress** cypress/included:7.7.0
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/run/desktop/mnt/host/c/Users/TAnderson2/source/repos/cypress/integration" to rootfs at "/root/.cache/Cypress/7.7.0/Cypress/Cypress" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type.
